projects/dsp-ui/src/lib/viewer/operations/display-edit/display-edit.component.ts

```diff
@@ -114,6 +121,48 @@ export class DisplayEditComponent implements OnInit {
         }
     }
 
+    openDialog() {
```

Comment:
Each method in TS file should have a JsDoc annotation to describe the method, e.g. resource-view component

Reply:
added JsDocs in 18faf9f